backend/CertificateManagerAPIs/DTO/CertificateDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace CertificateManagerAPIs.DTO
+{
+    public class CertificateDto
+    {
+
+        public SupplierDto Supplier { get; set; } = null!;
+        public string Type { get; set; } = null!;
```

Comment:
you have to decide will you have new line between properties or not, you can not mix